frontend/src/components/common/form/FastCurrencyInput.tsx

```diff
@@ -132,7 +132,7 @@ const CurrencyInput = ({
           required={required}
           placeholder={placeholder || ''}
         />
-        {!!tooltip && <TooltipIcon toolTipId="currency" toolTip={tooltip} />}
+        {!label && !!tooltip && <TooltipIcon toolTipId="currency" toolTip={tooltip} />}
```

Comment:
I'm not sure about this, is the label really a requirement of having a tooltip?

Reply:
It seems there was a bug in this component and if I dont put this condition here it was showing two tooltip icons one with label and another with currency textbox